<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramelessMainWindow</class>
 <widget class="QMainWindow" name="FramelessMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window Title</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QWidget#centralWidget {
background-color: rgba(250, 250, 250, 0.91);
border-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
border-radius:10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#titleWidget{
background-color: rgb(226, 231, 237);
color: rgb(0, 0, 0);
border-top-left-radius: 10px;
border-top-right-radius: 10px;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>277</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labTitle">
         <property name="text">
          <string>Window Title</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>299</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbtnMin">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: none;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);

	
	background-color: rgb(192, 192, 192);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/min-light.png</normaloff>:/icons/min-light.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtnMax">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: none;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(192, 192, 192);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/max-light.png</normaloff>:/icons/max-light.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtnClose">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	border-top-right-radius: 9px;
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: none;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	qproperty-icon: url (:/icons/close-dark.png);
	qproperty-iconSize: 11px 11px;
	background-color: rgb(180, 0, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/close-light.png</normaloff>:/icons/close-light.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bodyWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="tailWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#tailWidget{
background-color: rgb(226, 231, 237);
color: rgb(0, 0, 0);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labStatus">
         <property name="text">
          <string></string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>743</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
